radar/2024-10-1/bulcoin.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Blockchain State: Chuvashia - Volga Bulgaria"
+ring: develop
+quadrant: governance
+tags:
+  - governance
+  - services
+  - community
+  - DAO
+  - decentralized
+  - citizenship
+  - blockchain
+  - technological
+  - living
+  - residents
+  - BLC
+  - BulCoin
+  - Chuvashia
+  - VolgaBulgaria
+  - state
+---
+**Blockchain State: Chuvashia Volga-Bulgaria**. Our citizens [can elect representatives](https://t.me/bulcoin_blc/24) through the blockchain. Resolve conflicts in [private courts\(https://telegra.ph/Sudebnaya-sistema-v-CHuvashii---Volzhskoj-Bulgarii-07-20) with a blockchain reputation. The economy is supported by the [BulCoin-BLC](https://dedust.io/swap/TON/EQD_tFz9JQRVZNMgy3bzvnzei9deo9qc5NMtumG4_sx1zZRc), which is minted every month to the budget of the [elected Council\(https://x.com/ChuvashiaGov). The council uses the budget to expand the community. For example, through a [mini-app](https://t.me/bulcoin_blc/29) with freelance services. Freelance requires art, and this allows integrating the private court system. Citizens will be able to receive not only NFT passports, but also paper [foreign passports](https://t.me/bulcoin_blc/25).
```

Comment:
Another important point: the current description is written in the perfect tense, which creates the impression that everything described has already been developed and is fully operational. However, in reality, much of it is either still at the conceptual stage or under development.  

   The radar is not just a curated list — it also reflects the development history of a project. Therefore, it is important to represent the actual state of affairs and update the information over time. If information about a project appears in multiple versions (editions) of the radar, the project page will show its development history.  

The idea of creating a radar for L2S projects is not unique — it builds upon the concept of the [techradar by AOE](https://www.aoe.com/techradar/). This radar is focused on web development technologies. Here’s an example of a page with historical data: [aoe.com/techradar/languages-and-frameworks/angular/](https://www.aoe.com/techradar/languages-and-frameworks/angular/)